doc/tutorials/vuart_configuration.rst

```diff
@@ -88,6 +88,12 @@ connection. Or click **-** to delete a connection.
    :align: center
    :class: drop-shadow
 
+   .. note::
```

Comment:
This note needs to be outdented to the left margin (that will get rid of the doc build error), and I'd suggest this wording change:

```
.. note::  The release v3.0 ACRN Configurator assigns COM2 (I/O address ``0x2F8``) to the S5 feature.
   A conflict will occur if you assign ``0x2F8`` to another connection.  In our example, we'll
   use COM3 (I/O address ``0x3F8``).


```